### DISCUSSION TITLE
Battle #171 Pumpkin

### DISCUSSION THREAD

**Discussion by eightmillion** [cssbattle/forum#363] [General]
Can anyone explain why I can't hit 100% on this one? This is my code:

`<p k><p l><p st><p><p r><p j><style>body{background:#784972}p{background:#F7FFCF;width:100;height:150;border-radius:50%;position:fixed;top:74;left:140;border:11.5q solid#784972}[l]{left:100}[r]{left:180;z-index:-1}[k]{left:60}[j]{left:220;z-index:-2}[st]{width:22;height:80;border-radius:10vh;border:none;top:34;left:189;z-index:1`

The diff shows no difference. I even tried a previewer mentioned in another discussion and it looks exactly the same.

![Screenshot 2023-11-04 144119](https://github.com/cssbattle/forum/assets/164165/ec16233b-a8c6-4579-a89c-76e98722cb4c)

**Comment by Tryskall** [comment]
I changed the order of the p's to eliminate the Z-index and changed the border to outline, and finally adjusted the left on the [st] stem and got it to 100%. I realize you posted this over a year ago and might have figured it out, but i thought it would be a challenge to figure out your code. I managed to get my code down to 267 using padding instead of width and height.

<p><p j><p k><p l><p r><p st><style>
body{background:#784972}
p{background:#F7FFCF;width:100;height:150;border-radius:50%;position:fixed;top:84;left:70;outline:10px solid#784972}
[j]{left:230}
[k]{left:110}
[l]{left:190}
[r]{left:150}
[st]{width:22;height:80;border-radius:9in;outline:0;top:34.5;left:189
